.github/workflows/rector.yaml

```diff
@@ -43,15 +35,11 @@ jobs:
 
             -   run: composer install --no-progress --ansi
 
-            ## First run Rector - here can't be --dry-run !!! it would stop the job with it and not commit anything in the future
+            ## here can't be --dry-run, as the change results in error code 1 and stops the job without commiting in the future
             -   run: bin/rector process ${{ matrix.paths }} --ansi
 
-            -   run: vendor/bin/ecs check --fix --ansi
```

Comment:
cs fix still needed after apply rector

Reply:
This is to comply with dependency-less defaults, to make it easier to spread and use our own command to generate it on the fly :)
We'll use the coding standard in standalone run.